### DISCUSSION TITLE
how to zoom in and select mask areas

### DISCUSSION THREAD

**Discussion by mdkberry** [continue-revolution/sd-webui-segment-anything#145] [Q&A] [answered]
It keeps missing a part but the image is too small for me to accurately add a positive point to the mask

Is there a way to zoom into the image for better masking point precision?

**Comment by continue-revolution** [comment, marked as answer]
At this moment, no. In the future: possibly. I do have plan to support zoom-in feature in the near future, but I'm not sure how long it takes. I will soon release a refactored version of this extension, after that I will try to enable zoom-in just like what A1111 did for inpainting.

**Reply by ride5k** [reply]
this would be a very welcome change, if it is possible!
